databricks/koalas/groupby.py

```diff
@@ -640,6 +640,65 @@ def diff(self, periods=1):
             lambda sg: sg._kser._diff(periods, part_cols=sg._groupkeys_scols)
         )
 
+    def cumcount(self, ascending=True):
+        """
+        Number each item in each group from 0 to the length of that group - 1.
+
+        Essentially this is equivalent to
+
+        >>> self.apply(lambda x: pd.Series(np.arange(len(x)), x.index)) # doctest: +SKIP
```

Comment:
Could you add an expected result even if we skip the test?

Reply:
Why is this skipped, btw?

Reply:
ah, maybe:

```
        .. code-block:: python
            self.apply(lambda x: pd.Series(np.arange(len(x)), x.index))
```

as per pandas doc https://github.com/pandas-dev/pandas/blob/v1.1.0/pandas/core/groupby/groupby.py#L2199-L2256